extensionsGallery.json

```diff
@@ -1779,14 +1779,14 @@
 					},
 					"versions": [
 						{
-							"version": "1.34.0",
-							"lastUpdated": "12/14/2021",
+							"version": "1.35.0",
+							"lastUpdated": "2/23/2022",
```

Comment:
Use today's date for these - the lastUpdated should be the date the gallery was actually updated.